app/controllers/v2/link_sets_controller.rb

```diff
@@ -5,13 +5,12 @@ def get_links
     end
 
     def expanded_links
-      json = Rails.cache.fetch ['expanded-links', content_id, params[:with_drafts], params[:locale]], expires_in: 1.hour do
-        Queries::GetExpandedLinks.call(
-          content_id,
-          params[:locale],
-          with_drafts: with_drafts?,
-        )
-      end
+      json = Queries::GetExpandedLinks.call(
+        content_id,
+        locale,
+        with_drafts: with_drafts?,
+        generate: params.fetch(:generate, false),
```

Comment:
If the query is `?generate=false` the link expansion will be regenerated since `"false"` is truthy, might be worth clarifying this in the documentation or changing the logic here.